CRM/Civicase/Form/Report/BaseExtendedReport.php

```diff
@@ -854,7 +868,7 @@ protected function getSelectedDataFunctionField() {
    *
    * @return string
    */
-  function getTemplateFileName() {
+  public function getTemplateFileName() {
```

Comment:
This function is overridden and the base class has it set as public by default. So I had to make it public since I didn't want to make changes to base class.